eos-fix-home-dir-permissions

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+# Prior to 3.1.6, home directory permissions were created with dir mode 755.
+# This is now fixed, but on upgrade from an earlier version to 3.1.6,
+# let's perform a one-time clean up of any existing home dirs.
+
+stamp_file=/var/eos-dir-mode-700
```

Comment:
I see that I have a /var/lib/misc, but I don't know if it is guaranteed to exist.  To be on the safe side, I assume it is best to `mkdir -p` it.